### DISCUSSION TITLE
Small Issue with Visual Mode

### DISCUSSION THREAD

**Discussion by Valour549** [seerge/g-helper#3503] [General]
Version 0.199.0

I noticed a small bug with Visual Mode. So last time the **Disabled** option got added, and when it's selected and you change the color-temperature nothing happens (which is expected). But when you select a Gamut, the Visual Mode actually switches back on to the last visual mode used (whatever that was), but the selection itself is still stuck on Disabled.

It should either do nothing, or update to the correct visual mode. Doing nothing would be consistent with the behavior of the color-temperature selector.

Better yet, disable color-temperature and gamut selection completely when Disabled is selected. It will remind the user to actually select the mode they want~

**Comment by seerge** [comment]
@Valour549 hello, try this build. It should disable Color Gamut and Temp dropdowns when `Disabled` is selected
[GHelper.zip](https://github.com/user-attachments/files/18218370/GHelper.zip)

**Reply by seerge** [reply]
@Valour549 does it work for you ? :)

**Reply by Valour549** [reply]
@seerge Works great, thanks! BTW how did you get the Fn-Lock to work, I don't see it anywhere on my keyboard or inside AC, very cool. I also noticed that it only work whilst GH is running.

**Reply by seerge** [reply]
@Valour549 ok, great. It's live in 0.200 now.

FN-lock is pure software emulation. App just registers F1-F11 as own "hotkeys" in windows, and runs corresponding action when system notifies it about hotkey press.

**Reply by Valour549** [reply]
@seerge I checked out 0.200 and the new features. The High Performance power plan in control panel is only visible when I select High Performance Plan from Fans+Power.

What edits are done to make the High Performance power plan visible in Control Panel? I would like it to be permanently visible.

`powercfg -duplicatescheme 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c` Is it this or something else.

**Reply by seerge** [reply]
@Valour549 windows hides it when it's not selected, but that's same if you set it directly by GUID using `powercfg`. It's just how windows behaves.
